recipes/libheif/all/conanfile.py

```diff
@@ -13,9 +13,13 @@ class Libheif(ConanFile):
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
     options = {"shared": [True, False], "fPIC": [True, False],
-               "with_x265": [True, False]}
+               "with_x265": [True, False],
+               "with_dav1d": [True, False],
+               "with_libaomav1": [True, False]}
     default_options = {"shared": False, "fPIC": True,
-                       "with_x265": False}
+                       "with_x265": False,
+                       "with_dav1d": False,
+                       "with_libaomav1": False}
```

Comment:
Did you try with these options enabled? Is it robust?

Reply:
Sure, tried on mac, Linux and Windows